packages/amplication-client/src/Assistant/OnboardingWithJovu.tsx

```diff
@@ -35,7 +40,11 @@ const OnboardingWithJovu: React.FC = () => {
   const handleSubmit = useCallback(
     (message: string) => {
       sendOnboardingMessage(message);
+      expireCookie("signup");
```

Comment:
@yuval-hazaz  Why??...

Reply:
to keep the user away from the onboarding once they submitted the prompt once